docs/templates/getting-started/bounding_boxes.md

```diff
@@ -53,7 +53,7 @@ box_data = np.array(
 
 Let's create a simple visualization pipeline.
 For an explanation of what control-map is doing please check our tutorial at:
-[paz/examples/tutorials/controlmap_processor.py](paz/examples/tutorials/controlmap_processor.py)
+[paz/examples/tutorials/controlmap_processor.py](https://github.com/oarriaga/paz/blob/master/examples/tutorials/controlmap_processor.py)
```

Comment:
Thank you for the PR! Could you please make it a hyperlink? Similar to how it's done on [line 5](https://github.com/oarriaga/paz/pull/334/files#diff-e16aa4cad27040e2d819d645e4a93add7d9efafe03b4942d6dcfa52757e0deafR5)

Reply:
Hi @proneetsharma !

Sorry, but I don't understand your request. Could you please explain it? I think I did what you're asking. I created an absolute hyperlink like on line 5.

Reply:
I mean instead of file path you can make with a text, something like this:
For an explanation of what control-map is doing please check our tutorial [here](https://github.com/oarriaga/paz/blob/master/examples/tutorials/controlmap_processor.py).

Reply:
Looks good. Thank you!

Reply:
Thank you!